tests/functional/olp-cpp-sdk-authentication/AuthenticationClientTest.cpp

```diff
@@ -173,6 +173,11 @@ class AuthenticationClientTest : public AuthenticationCommonTestFixture {
 
     return request_future.get();
   }
+
+  std::string PrintErrorId(
```

Comment:
```suggestion
  std::string GetErrorId(
```
As it is technically not printing anything but only returning.